cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Frequently Asked Questions
+summary: Get answers to frequently asked questions about CockroachCloud Free (beta)
+toc: true
+redirect_from:
+- ../stable/cockroachcloud-frequently-asked-questions.html
+---
+
+This page answers the frequently asked questions about CockroachCloud Free (beta). For answers to frequently asked questions about the paid version of CockroachCloud, see [CockroachCloud FAQs](frequently-asked-questions.html).
+
+## General
+
+### What is CockroachCloud Free?
+
+CockroachCloud Free delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that also removes the friction of initial cluster sizing and auto-scales based on your application traffic. While we have eliminated the concept of nodes for you, there will be an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster.
+
+### Do I have to pay for CockroachCloud Free?
+
+CockroachCloud Free is free forever.
```

Comment:
IMO we should add a line that answers the question more directly, something like "No, you do not have to pay anything.  CockroachCloud is free forever (tm)"